### ISSUE TITLE
Issue with forced keyboard trap method

### ISSUE THREAD

**Issue by Tonylayton1** [Section508Coordinators/DHSA-TT-200-A#245]
Update the keyboard trap failure to prevent reverse tabbing up and out beyond the First Name field. Implement this the same way as in the incremental. Lastly, add r3e03 to the list of failure reasons for 4.C on this page.

**Comment by mikebarklage** [comment]
Done.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by AnnMarieDavis** [comment]
Reverse tabbing is fixed and students cannot exit the trap.  However, when students reverse tab through the page, they go from whichever device type they have selected and skip over all the address elements = going directly the first name text box.

**Comment by mikebarklage** [comment]
I was curious and checked - this also happens on incremental page 3. Reverse tabbing from the City field skips up to Contact Name.

I'm not sure how to fix this...

**Comment by Tonylayton1** [comment]
When the user is tabbing backwards and enters into keyboard trap area, focus moves to the first item in the keyboard trap rather than starting at the bottom. 

The same is true for the keyboard trap example we found here and used for the webpage: https://accessibility.18f.gov/keyboard/

Should we try and change the way the keyboard trap works, or can we update the answer sheet to accommodate that behavior?

**Comment by drewnielson** [comment]
As discussed during our meeting yesterday, let's keep as is for now. We'll just accept the behavior (landing on the first element within the keyboard trap when tabbing backward into the keyboard trap) as a consequence of the keyboard trap without adding it to the list of logical focus failures.

**Comment by AnnMarieDavis** [comment]
Since we agreed to the behavior, this has already been verified.  👍
